src/components/Navbar/Navbar.js

```diff
@@ -8,13 +8,25 @@ import {
   faBars,
   faTimes
 } from '@fortawesome/free-solid-svg-icons';
+
 import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 
-import {ACCOUNT, MY_BOOK_HISTORY} from '../../routes';
+import * as ROUTES from '../../routes';
 
+import {AuthUserContext} from '../Session/Session';
 import Button from '../Button/Button';
+import SignOut from '../SignOut/SignOut';
+
+const NavbarAuth = () => <SignOut />;
 
-class NavBar extends React.Component {
+const NavbarNonAuth = () => (
+  <React.Fragment>
+    <Link to={ROUTES.LOG_IN}>Sign in</Link>;
```

Comment:
As SignIn and SignUp are integrated into one component in the design, we only need one React Component for them as well. But we can change that later.

Reply:
Sure, we can change this later!